src/topp/DatabaseSuitability.cpp

```diff
@@ -135,17 +134,17 @@ class DatabaseSuitability :
   // it gets automatically called on tool execution
   void registerOptionsAndFlags_() override
   {
-    registerInputFile_("in_id", "<file>", "", "Input idXML file from peptide search with combined database with added de novo peptide (after FDR)");
+    registerInputFile_("in_id", "<file>", "", "Input idXML file from peptide search with combined database with added de novo peptide. PeptideIndexer is needed, FDR is forbidden.");
     setValidFormats_("in_id", { "idXML" });
     registerInputFile_("in_spec", "<file>", "", "Input MzML file used for the peptide identification");
     setValidFormats_("in_spec", { "mzML" });
-    registerInputFile_("in_novo", "<file>", "", "Input idXML file containing de novo peptides");
+    registerInputFile_("in_novo", "<file>", "", "Input idXML file containing de novo peptides (unfiltered)");
     setValidFormats_("in_novo", { "idXML" });
     registerOutputFile_("out", "<file>", "", "Optional tsv output containing database suitability information as well as spectral quality.", false);
     setValidFormats_("out", { "tsv" });
-    registerDoubleOption_("novor_fract", "<double>", 1, "Set the fraction of how many cases, where a de novo peptide scores just higher than the database peptide, you wish to re-rank.", false, true);
-    setMinFloat_("novor_fract", 0);
-    setMaxFloat_("novor_fract", 1);
+    registerDoubleOption_("cut_off_fract", "<double>", 1, "Percentile to determine which decoy cut-off to use. '1' use the highest cut-off to '0' use the lowest one.", false, true);
```

Comment:
is `1` a good default?

Reply:
should be, right?
'1' results in the lowest difference being used as a cut-off. This is the most strict and it's guaranteed to work.

Reply:
Yes it is guaranteed to work, but does it produce nice results on average?
What do you use for benchmarking?

Reply:
For benchmarking 0.99 was used because that's the setting from the original paper and we wanted comparable results. What would produce the nicest results on average wasn't tested. That's probably not a bad idea though...